### ISSUE TITLE
BO - Product edit (SEO section)-  Remaining characters counter should be counted real time

### ISSUE THREAD

**Issue by ShaiMagal** [PrestaShop/PrestaShop#37095]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

If eshop owner is editing in product  Meta title (or Meta description), there si character counter. But this character counter isn't changing while eshop owner is typing.

![Untitled](https://github.com/user-attachments/assets/2a9f1ff8-d2dd-4a11-87cf-7d64c6460786)

### Expected behavior

Character counter should be changed while eshop owner is typing real time.

### Steps to reproduce

1) go to product edit
2) open tab "SEO"
3) write something to meta title
4) character counter on right side isn't changing

### PrestaShop version(s) where the bug happened

8.2.0

### PHP version(s) where the bug happened

8.1

### Your company or customer's name goes here (if applicable).

https://www.openservis.cz/

**Comment by Hlavtox** [comment]
True, just reproduced on 8.1.7, I never noticed.

Works on category page, it's using a different input.

**Comment by ShaiMagal** [comment]
@Hlavtox yes, there is something wrong, I think, we should fix it for PS8.2 branch as well.

**Comment by Progi1984** [comment]
Fixed by #38749
